<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
        <widget class="QGraphicsView" name="image_viewer_1"/>
    </item>
    <item row="1" column="1">
        <widget class="QSlider" name="image_slider_0">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="image_slider_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
        <widget class="QSlider" name="image_slider_1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGraphicsView" name="image_viewer_2"/>
    </item>
    <item row="0" column="1">
        <widget class="QGraphicsView" name="image_viewer_0"/>
    </item>
       <item row="2" column="1">
           <layout class="QGridLayout" name="grid_layout">
               <item row="3" column="1">
                   <widget class="QLabel" name="image_contrst_max_label">
                       <property name="text">
                           <string>Change contrast (max)</string>
                       </property>
                   </widget>
               </item>
               <item row="2" column="2">
                   <widget class="QSlider" name="image_contrast_min_slider">
                       <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                       </property>
                   </widget>
               </item>
               <item row="3" column="2">
                   <widget class="QSlider" name="image_contrast_max_slider">
                       <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                       </property>
                   </widget>
               </item>
               <item row="2" column="1">
                   <widget class="QLabel" name="image_contrast_min_label">
                       <property name="text">
                           <string>Change contrast (min)</string>
                       </property>
                   </widget>
               </item>
               <item row="1" column="1">
                   <widget class="QLabel" name="image_cycle_label">
                       <property name="text">
                           <string>Cycle images</string>
                       </property>
                   </widget>
               </item>
               <item row="1" column="2">
                   <widget class="QSlider" name="image_cycle_slider">
                       <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                       </property>
                   </widget>
               </item>
           </layout>
       </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1273</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
